tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -126,8 +124,19 @@ def ldap_usergroup_name():
         user_groups[0].delete()
 
 
+@fixture()
```

Comment:
as far as I see in documentation, default scope for fixtures is a function. That means, if you run 4 or 8 tests in parallel, you can delete all ldaps from other tests in run-time. And that can consequently fail all of them

Reply:
All ldap tests always run in one thread as we won't have multiple ldap servers to test. https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_ldap_authentication.py#L44

Reply:
@ntkathole thanks , however it would be much great to set it `@fixture(scope='module')` as that will help to fix the scope of fixture too.

Reply:
Fixed!